src/components/lists/ArticleList/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from '../lists.module.scss';
+import List from '../List/index';
+import reduce from 'lodash/reduce';
+
+function ArticleList ( props ) {
+  const {
+    data,
+    type,
+    mapProps,
+  } = props;
+
+  const defaultItemProps = [ 'id', 'link', 'imageSrc', 'header', 'subHeader' ];
+  const itemProps = reduce(defaultItemProps, (res, val) => {
+    res[val] = mapProps[val] || val;
+
+    return res;
+  }, {});
+
+  return (
+    <List type={ type } >
```

Comment:
Hmm yeah it was like this originally because we were planning to expose it as a wrapper component to the end user, but in the current iteration I agree it can just be a `ul` - I'll make the change